### PR TITLE
Fix show_excerpts in collection layout's front matter

### PR DESCRIPTION
**Pull request by CrazyOrr**
This is a bug fix.
## Summary
The `show_excerpts` front matter in [collection layout](https://mmistakes.github.io/minimal-mistakes/docs/layouts/#layout-collection) doesn't work as documented for now. Fixed it.

### CODE REVIEW

**Comment by iBug** [comment]
not stale

File: _includes/archive-single.html [hunk 1]
```diff
@@ -10,6 +10,12 @@
   {% assign title = post.title %}
 {% endif %}
 
+{% if show_excerpts == false %}
+  {% assign show_excerpt = false %}
```
Comment (iBug) [inline _includes/archive-single.html:14]: Can we be consistent with plural form here? I'd go with `show_excerpts`. Same for 2 more occurrences below.

**Review by iBug — CHANGES_REQUESTED** [review]

File: _includes/archive-single.html (hunk 1 above)
Comment (CrazyOrr) [reply _includes/archive-single.html:14]: Sure, as long as you don't mind overshadowing the original variable `show_excerpts`.

Comment (iBug) [reply _includes/archive-single.html:14]: Hmmm, looks like there's no need for `show_excerpt` at all. Just use `show_excerpts != false` below and it'll be fine.

To be honest, I'm kind of puzzled about all these variables now.

Comment (CrazyOrr) [reply _includes/archive-single.html:14]: Yes, good idea.

**Pull request closed without merging**